### PR TITLE
make virtualhost:443 match formatting and entries of virtualhost:80

### PR DESCRIPTION
**Pull request by Takadonet**

### CODE REVIEW

File: templates/ngs.conf.erb [hunk 1]
```diff
@@ -17,17 +17,22 @@
...
+	</Location>
+
 	# REST API:
 	# IRIDA REST API (deployed to $TOMCAT_HOME/webapps/ as irida-web.war):
 	<Location /<%= @irida_url_path %>>
-	    ProxyPass http://<%= @irida_ip_addr %>:8080/<%= @irida_url_path %> connectiontimeout=7200 timeout=7200
+		ProxyPass http://<%= @irida_ip_addr %>:8080/<%= @irida_url_path %> connectiontimeout=7200 timeout=7200
 	</Location>
 
 	# Custom error pages
```
Comment (ericenns) [inline templates/ngs.conf.erb]: Please remove this duplicate error pages section.

**Review by ericenns — CHANGES_REQUESTED** [review]
One minor change requested, otherwise the two virtualhosts look near identical minus ssl configuration of course.

File: templates/ngs.conf.erb (hunk 1 above)
Comment (Takadonet) [reply templates/ngs.conf.erb]: removed

**Review by ericenns — APPROVED** [review]
👍 looks great now, thanks

**Pull request merged**